controllers/vehicle/curt.go

```diff
@@ -42,8 +42,14 @@ func CurtLookup(w http.ResponseWriter, r *http.Request, enc encoding.Encoder, dt
 	if hdstr == "true" {
 		heavyduty = true
 	}
+	// determine if you are going to get customer prices for each part
+	var getCustomerPrices bool
+	custPricingStr := r.URL.Query().Get("customerPrices")
+	if custPricingStr == "true" || custPricingStr == "True" {
```

Comment:
can you just do a custPricingStr.toLower() == "true" just so we're not case-sensitive at all.